spec/index.html

```diff
@@ -10369,7 +10369,151 @@ <h3>Evaluation Semantics</h3>
+              </p>
+              <div class="defn">
+                <b>Definition: <span id="defn_projmap" name="defn_projmap">Projection Expression Variable Remapping</span></b>
+                <p>
+                  For a projection algebra operation #sparqlProjection `Project(A, PV)` acting on algreg express `A` and with set of variables `PV`, define
+                  a partial mapping `F` from
+                  `<a href="#sparqlQueryVariables">V</a>`,
+                  the set of all variables, to `V` where:
+                </p>
+                <pre>F(v) = v1 if v is in PV, where v1 is a fresh variable
+F(v) = v if v is not in PV</pre>
+                <p>
+                Define the Projection Expression Variable Remapping `ProjectMap(P,PV)`
+                </p>
+                <pre>ProjectMap(Project(A, PV)) = Project(A1, PV) 
+                                where A1 is the result of applying F
+                                to every variable mentioned in A.
+                </pre>
+                <p>
+                  The Projection Expression Variable Remapping yields an algrebra expression that
+                  evalautes to the same results as the Project argument. No variable of `ProjectMap(Project(A, PV))`
+                  that is not in `PV` is mentioned anywhere else in the algebra expression for the query.
+                </p>
+              </div>
+            
+              <p>This process is applied throughout the graph pattern of <code>EXISTS</code>:</p>
+
+              <div class="defn">
+                <b>Definition: <span id="defn_varrename" name="defn_varrename">Variable Remapping</span></b>
+                <p>
+                  For any algebra expression `X`, define the Variable Remapping `PrjMap(X)`of algebra expression `X`:
+                </p>
+                  <pre>PrjMap(X) = replace all project operations Project(P, PV) 
+                with ProjectMap(P,PV) for each projection in X.</pre>
+              </div>
+              <p>
+                The outcome of `PrjMap` is independent of the order of replacement
+                (e.g. bottom-up or top-down).
+                Replacements may happen several times, depending on recusive order
+                but each time a replacement is made, the variable not used anywhere else.
+              </p>
+        
+              <p class="note">
+                A variable inside a project expression that is not in the variables projected
+                is not affected by the values insertion operation because it is renamed apart.
+              </p>
+            </section>
+
+            <section>
+              <h4>Values Insertion</h4>
+              <div class="defn">
+                <div>
+                  <b>Definition: <span id="defn_valuesinsertion" name="defn_valuesinsertion">Values Insertion</span></b>
+                  <p>
+                    Define the Values Insertion function `ValuesInsert(X, μ)`
```

Comment:
If, as I suggested above, `Values Insertion` is being changed to `Value Insertion`, then `ValuesInsert` should likewise be changed to `ValueInsert`
```suggestion
                    Define the Value Insertion function `ValueInsert(X, μ)`
```

Reply:
While this is in algebra space, the syntax analogy is `VALUES`. 

The μ can have several bindings, correlating several variables.

Reply:
Trying `InsertValues `.

This text is in "alternative 2" (runtime algebra modification) and the current direction is alternative 1 (translate time and a new algebra operator to get the correlated solution.